coverage/src/test/java/gov/cms/ab2d/coverage/service/CoverageServiceImplTest.java

```diff
@@ -212,7 +218,7 @@ void getOrCreateInsertsNew() {
         coveragePeriodRepo.delete(period);
 
         assertThrows(EntityNotFoundException.class,
-            () -> coverageService.getCoveragePeriod(contract1, 12, 2020)
+                () -> coverageService.getCoveragePeriod(contract1, 12, 2020)
```

Comment:
There might be an extra indentation here. Is this needed? I'm just thinking it may cause the linter to fail.

Reply:
reverted